WordPress/gutenberg
# [RNMobile] Enable the native editor to receive a host app's namespace

## Related PRs

* `Gutenberg Mobile`: https://github.com/wordpress-mobile/gutenberg-mobile/pull/5040 

## What?

This PR enables the native editor to receive a host app's namespace (e.g. "WordPress").

## Why?

Third-party host app's may wish to customize the native editor's appearance or behaviour, which is a challenge at the moment. Enabling the editor to receive a host app's namespace is one step towards addressing this challenge.

## How?

A new `hostAppNamespace` prop has been added, which can received from a host app via the React Native bridge.

## Testing Instructions

Installable builds, containing the changes from this PR, for both the iOS and Android apps can be found in the following PRs:

* `iOS`: https://github.com/wordpress-mobile/WordPress-iOS/pull/19084
* `Android`: https://github.com/wordpress-mobile/WordPress-Android/pull/16935

To help test the changes from this PR,  please open the editor in the WordPress app for both platforms and confirm that there are no regressions to the writing flow. These changes should cause no regressions, only allowing host app's to send their namespace.
